Classes/CardIOContext.m

```diff
@@ -38,6 +38,8 @@ - (void)set##prop_uc:(t)prop_lc { \
 CONFIG_PASSTHROUGH_READWRITE(BOOL, useCardIOLogo, UseCardIOLogo)
 CONFIG_PASSTHROUGH_READWRITE(UIColor *, guideColor, GuideColor)
 CONFIG_PASSTHROUGH_READWRITE(BOOL, allowFreelyRotatingCardGuide, AllowFreelyRotatingCardGuide)
+CONFIG_PASSTHROUGH_READWRITE(CGFloat, scannedImageDuration, ScannedImageDuration)
```

Comment:
Again, for more consistency that we've always followed in the past, could you swap the new line up above the `allowFreelyRotatingCardGuide` line?